### ISSUE TITLE
Realm Graphs

### ISSUE THREAD

**Issue by lightsighter** [StanfordLegion/legion#625]
This is our issue for tracking progress on support for Realm graphs. Realm graphs will further accelerate our ability to replay traces of Realm operations efficiently by memoizing chunks of a Realm graph and giving them efficiently replay.

**Comment by lightsighter** [comment]
Once Realm Graphs are stable and ready for use please assign this issue back to @lightsighter and @magnatelee so we can modify the tracing infrastructure to target them.

**Comment by streichler** [comment]
Realm graphs are already in a branch and waiting on tracing infrastructure to have anything other than taskbench target them.  Ideally we're driving optimizations based on quantifiable needs, and we don't know what traced Legion/Regent applications need yet.

**Comment by elliottslaughter** [comment]
FYI @seemamirch is starting some of the work for Legion tracing, I think she is the appropriate next point of contact.

**Comment by lightsighter** [comment]
I agree, removing @magnatelee and myself and assigning @seemamirch

**Comment by streichler** [comment]
This issue has been superseded by #792.
